### ISSUE TITLE
Loss Function SparseCategoricalCrossentropy not implmented

### ISSUE THREAD

**Issue by ARadauer** [deeplearning4j/deeplearning4j#9468]
If i use the SparseCategoricalCrossentropy loss function in with keras in phyton:

```
model.compile(
  optimizer='adam',
  loss=tf.losses.SparseCategoricalCrossentropy(from_logits=True),
  metrics=['accuracy'])
```

And use the saved model as h5 file in java with deeplearning4j

`MultiLayerNetwork model = KerasModelImport.importKerasSequentialModelAndWeights("ki.h5");`

I get the exception:

`Unknown Keras loss function sparsecategoricalcrossentropy. Please file an issue at https://github.com/eclipse/deeplearning4j/issues.`

I use the deeplearning4j version 1.0.0-M1.1

I would be very grateful for a workaround or a a fix because other loss functions did not perform very well with my usecase

**Comment by ARadauer** [comment]
@agibsonccc This is the issue regarding my question on stackoverflow https://stackoverflow.com/questions/69420171/use-in-python-trained-tensorflow-keras-model-in-java-for-image-recognition

Thank oyu

**Comment by agibsonccc** [comment]
@ARadauer of note here is it appears we have mapped this: https://github.com/eclipse/deeplearning4j/blob/75899208427bb755f8629d6a17d594162a078fa4/deeplearning4j/deeplearning4j-modelimport/src/main/java/org/deeplearning4j/nn/modelimport/keras/config/KerasLayerConfiguration.java#L359 - you can specify it as sparse_categorical_crossentropy.
 Do you have the associated model I can look at? It might be the way the loss function is showing up in the configuration where we're missing something.

**Comment by agibsonccc** [comment]
@ARadauer closing this since you're not responding. I'll assume you've moved on. If you ever get time to help me reproduce this let me know.
